lib/hydra/works/models/concerns/file.rb

```diff
@@ -0,0 +1,12 @@
+module Hydra::Works
+  # Namespace for Modules with functionality specific to Files
+  module File
```

Comment:
I'm not sure I like this. It destroys the parallels between Hydra::PCDM::File, which is a class.